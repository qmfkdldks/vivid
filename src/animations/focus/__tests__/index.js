import React from "react";
import { shallow } from "enzyme";
import Focus from "../index";

describe("Focus", () => {
  it("should render successfully", () => {
    const component = shallow(<Focus>Hide and sneak</Focus>);
    expect(component).toMatchSnapshot();
  });
});
