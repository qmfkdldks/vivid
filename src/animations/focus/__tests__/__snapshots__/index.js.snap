// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Focus should render successfully 1`] = `
<Focus>
  Hide and sneak
</Focus>
`;
